src/mame/drivers/plygonet.cpp

```diff
@@ -418,13 +430,13 @@ void polygonet_state::main_map(address_map &map)
 	map(0x000000, 0x1fffff).rom();
 	map(0x200000, 0x21ffff).ram().w(m_palette, FUNC(palette_device::write32)).share("palette");
 	map(0x400000, 0x40001f).rw(m_k053936, FUNC(k053936_device::ctrl_r), FUNC(k053936_device::ctrl_w));
-	map(0x440000, 0x440fff).rw(FUNC(polygonet_state::polygonet_roz_ram_r), FUNC(polygonet_state::polygonet_roz_ram_w));
-	map(0x480000, 0x480003).r(FUNC(polygonet_state::polygonet_inputs_r));
-	map(0x4c0000, 0x4c0003).w(FUNC(polygonet_state::polygonet_sys_w));
-	map(0x500000, 0x503fff).ram().w(FUNC(polygonet_state::shared_ram_write)).share("shared_ram");
+	map(0x440000, 0x440fff).rw(FUNC(polygonet_state::roz_ram_r), FUNC(polygonet_state::roz_ram_w));
```

Comment:
Same for this one – you could use a `required_shared_ptr` to access the memory.

Reply:
Done.